### DISCUSSION TITLE
Playlist Weights

### DISCUSSION THREAD

**Discussion by dekakast** [AzuraCast/AzuraCast#5744] [Q&A]
Is there a discernable difference between a playlist weight of 1 vs. 2, and playlist weight of 4 vs. 5?

**Comment by Mozitux** [comment]
Hi, here is an example for you to understand:
If playlist A has a weight of 3 and it overlaps on the schedule with playlist B which has a weight of 8, then playlist B will be played first.

**Reply by robz0** [reply]
That's useful thanks @Mozitux . i thought 1 was 'higher' than 2, as in priority or race positioning. So it is the opposite, which implies the help line on the playlist edit screen is ambiguous. Unless thinking weight as in Kilos :)
